### ISSUE TITLE
training with custom dataset

### ISSUE THREAD

**Issue by mhyeonsoo** [stevliu/self-conditioned-gan#7]
Hello,

Thanks for the great idea.
I am now trying to train the model with my own dataset which has 1 class.
Can you briefly guide me how to / what to modify for training own dataset?

What I've changed is,
```
1. edited config by coping imagenet configs and changing number of classes / name in it.
2. added class for loading my own dataset in inputs.py script
```

Any other things should be required?

thank you

**Comment by stevliu** [comment]
Yes, changing the config by changing data['type'] and changing the number of classes to 1 should do it. 

The class that you write should be a torch.data.Dataset class.

**Comment by woshichunge12** [comment]
@mhyeonsoo 
How did you do it? Did you train successfully later?thanks a lot.
